README.md

```
@@ -23,3 +23,38 @@ make dev
+  update the database.
+
+Running:
+
+1.
+    ```shell
+    npm install
+    node contrib/whats-extract.js
+    ```
+
+2. Copy each output and paste it into a new terminal. It should look like this:
+    ```shell
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)